go.mod

```diff
@@ -11,12 +11,12 @@ require (
 	cosmossdk.io/log v1.2.1
 	cosmossdk.io/math v1.1.3-rc.1
 	cosmossdk.io/store v1.0.0-rc.0
+	cosmossdk.io/x/gov v0.0.0-00010101000000-000000000000
```

Comment:
Is it only because of non extracted modules or the sdk imports gov somewhere else?

Reply:
one or two places for testing but they will be removed as well